aspirateur/tests/textes/test_get_dossiers_legislatifs.py

```diff
@@ -0,0 +1,96 @@
+import os
+from datetime import date
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+
+HERE = Path(os.path.dirname(__file__))
+
+
+@pytest.yield_fixture
+def sample_file():
+    path = HERE / ".." / "sample_data" / "Dossiers_Legislatifs_XV.json"
+    with open(path) as f_:
```

Comment:
`yield path.open()` should be enough.

Reply:
I wanted to make sure the file was properly closed after use by using a context manager here, but I'm just mocking the opening of the file here, and it will be closed in the `get_dossiers_legislatifs` function, so you're right we can make it simpler, and just a `return path.open()` will even do.

Fixed in 09c6821bc2fba4991a3cdfa79b81b41223aa7c32